### ISSUE TITLE
Standalone actions prevent using non-keyword parameters in `#initialize` (v2)

### ISSUE THREAD

**Issue by tomdalling** [hanami/controller#330]
I just got tripped up by this:

```ruby
class MyAction < Hanami::Action
  def initialize(arg)
    @arg = arg
  end

  def handle(*)
  end
end

MyAction.new(5)
# !!! ArgumentError: wrong number of arguments (given 1, expected 0)
#     from [...]/lib/hanami/action/standalone_action.rb:262:in `new'
```

The exception message is confusing because it sounds like `.new` expected zero arguments.

This is caused by `StandaloneAction.new` only forwarding keyword arguments to `#initialize`. Could it forward positional and block args too? I tested it quickly and it seemed ok.

Let me know if you want a PR.

**Comment by jodosha** [comment]
@timriley What do you think?

**Comment by timriley** [comment]
I think this makes sense! I reckon go ahead, @tomdalling — thanks!

**Comment by tomdalling** [comment]
Closed by #331
